HLTrigger/Muon/src/HLTScoutingMuonProducer.cc

```diff
@@ -124,7 +193,9 @@ void HLTScoutingMuonProducer::fillDescriptions(edm::ConfigurationDescriptions& d
     desc.add<edm::InputTag>("HcalPFClusterIsoMap", edm::InputTag("hltMuonHcalPFClusterIsoForMuons"));
     desc.add<edm::InputTag>("TrackIsoMap", edm::InputTag(
                                 "hltMuonTkRelIsolationCut0p09Map:combinedRelativeIsoDeposits"));
+    desc.add<edm::InputTag>("displacedvertexCollection", edm::InputTag("hltDisplacedmumuVtxProducerDoubleMu3NoVtx"));
     desc.add<double>("muonPtCut", 4.0);
```

Comment:
The collection ```hltDisplacedmumuVtxProducerDoubleMu3NoVtx``` does not have a producer in the menu.
So is this a mistake or just a dummy placeholder?